### PR TITLE
Add image metadata delete endpoint

### LINKED ISSUE #5: Add Image Metadata Remover
There should be an endpoint that allows users to upload their pictures with metadata, and receive a copy of their picture with the metadata removed.

### PR DESCRIPTION
**Pull request by penguingovernor**
As of right now this endpoint deletes the image metadata. 

In a future pull request, we can create a new image with randomized metadata using [sharp's `withMetadata` API](https://sharp.pixelplumbing.com/api-output#withmetadata).

For the time being this should close #5

### CODE REVIEW

File: backend/amplify/backend/api/scrubber/scrubber-cloudformation-template.json [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+            "Type":  "String"
+        },
+        "unauthRoleName": {
+            "Type":  "String"
+        },
+        "env": {
+            "Type": "String"
+        },
+    
+
```
Comment (ianjennings) [inline backend/amplify/backend/api/scrubber/scrubber-cloudformation-template.json:16]: This spacing is awkward. Doesn't matter for a hackathon though.

File: backend/amplify/backend/backend-config.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+					"category": "function",
+					"resourceName": "scrubFunc",
+					"attributes": [
+						"Name",
+						"Arn"
+					]
+				}
+			]
+		}
+	}
```
Comment (ianjennings) [inline backend/amplify/backend/backend-config.json:24]: Consider configuring your editor to use 2 or 4 spaces.

File: backend/amplify/backend/function/scrubFunc/src/app.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2017 - 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```
Comment (ianjennings) [inline backend/amplify/backend/function/scrubFunc/src/app.js]: This is probably copy and paste, but this line gives Amazon right to all your code!

File: backend/amplify/backend/function/scrubFunc/src/app.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the License. A copy of the License is located at
+    http://aws.amazon.com/apache2.0/
+or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+*/
+
+var express = require("express");
+var bodyParser = require("body-parser");
+var sharp = require("sharp");
+var awsServerlessExpressMiddleware = require(
```
Comment (ianjennings) [inline backend/amplify/backend/function/scrubFunc/src/app.js]: ES6 supports some new (better?) ways to do require. `var` is fine for hackathons.
https://www.freecodecamp.org/news/requiring-modules-in-node-js-everything-you-need-to-know-e7fbd119be8/

File: backend/amplify/backend/function/scrubFunc/src/app.js [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+      ".jif",
+      ".jfif",
+      ".jfi",
+      ".tif",
+      ".tiff",
+    ],
+  );
+  const lowerCaseExt = String(ext).toLowerCase();
+  const lowerCaseExtWDot = "." + lowerCaseExt;
+
```
Comment (ianjennings) [inline backend/amplify/backend/function/scrubFunc/src/app.js:38]: Oh yeah there you go.

File: backend/amplify/backend/function/scrubFunc/src/app.js [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+app.post("/api/v001/scrub", async function (req, res) {
+  // Get the data and the extension off
+  // the body.
+  const { data, ext } = req.body;
+
+  // Ensure they're both defined.
+  if (data === undefined || ext === undefined) {
+    res.status(400).json({
+      "success": false,
+      "message": "missing required field",
```
Comment (ianjennings) [inline backend/amplify/backend/function/scrubFunc/src/app.js]: Would be good to mention which field is missing here.

File: backend/amplify/backend/function/scrubFunc/src/app.js [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+      "data": "none",
+    });
+    return;
+  }
+
+  // Validate the base64 payload.
+  //
+  // Source: https://stackoverflow.com/questions/475074/regex-to-parse-or-validate-base64-data
```
Comment (ianjennings) [inline backend/amplify/backend/function/scrubFunc/src/app.js:67]: Excellent

File: backend/amplify/backend/function/scrubFunc/src/app.js [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+  let err = null;
+
+  try {
+    imgBuf = await image.toBuffer();
+  } catch (error) {
+    err = error;
+  }
+
+  // If not successful, bail.
+  if (err && !imgBuf) {
```
Comment (ianjennings) [inline backend/amplify/backend/function/scrubFunc/src/app.js]: I think just `err` is enough.

Comment (ianjennings) [reply backend/amplify/backend/function/scrubFunc/src/app.js]: Oh, maybe we want `err || !imgBuf`

**Review by ianjennings — APPROVED** [review]
LGTM

**Review by nicoledanuwidjaja — COMMENTED** [review]
Great documentation!

**Review by ianjennings — APPROVED** [review]
Awesome

**Review by ianjennings — APPROVED** [review]
Awesome

**Pull request merged**